tests/benchmarks/power_consumption/qdec/sysbuild.cmake

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Add radio core project
+ExternalZephyrProject_Add(
```

Comment:
do this properly as per https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/Kconfig.netcore#L15 and https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/netcore.cmake#L10